tests/test_oauth2client.py

```diff
@@ -144,6 +146,23 @@ def get(self, key, default=None):
     return Info(self._headers)
 
 
+@contextlib.contextmanager
+def mock_module_import(module):
+  """Place a dummy object in sys.modules to mock an import test"""
+  parts = module.split('.')
+  for n in range(len(parts)):
+    path = '.'.join(parts[:n+1])
+    sys.modules[path] = object()
+  
+  try:
```

Comment:
To prevent exceptions from causing leaks into sys.modules. See [here](https://docs.python.org/2/library/contextlib.html#contextlib.closing)